hack/test_k8s_e2e.sh

```diff
@@ -16,22 +16,37 @@
 
 set -e
 
-KUBECONFIG=${KUBECONFIG:-""}
+REPO_ROOT=$(realpath $(dirname ${BASH_SOURCE})/..)
 
-if [ -z "$KUBECONFIG" ]
-echo "KUBECONFIG not set"
-exit 1
+# Check KUBECONFIG env var is set or not
+# If not, then check if kubeconfig is under cloud-provider-azure repo's root
+# If still not, then exist 1
+if [ -z "$KUBECONFIG" ]; then
+  if [ -f "${REPO_ROOT}/kubernetes" ]; then
+    export KUBECONFIG="${REPO_ROOT}/kubeconfig"
+  else
+    echo "KUBECONFIG not set"
+    exit 1
+  fi
```

Comment:
if cluster is provisioned using aks-engine, kubeconfig can be found under REPOROOT/_output/dnsprefix/kubeconfig/xxxx.json. 
I think we don't need to set KUBECONFIG here.

Reply:
Actually this change is for tests migration to capz.
If `test_k8s_e2e.sh` is used by aks-engine then I need to reconsider the change to fix both.

Reply:
I decided to simply add a new script for capz because anyway the one for aks-engine will be deprecated. I think there's unnecessary to bother adding options or env var checks in original script.